src/pymovements/dataset/dataset_files.py

```diff
@@ -41,7 +41,7 @@
 from pymovements.reading_measures import ReadingMeasures
 
 
-def scan_dataset(definition: DatasetDefinition, paths: DatasetPaths) -> pl.DataFrame:
+def scan_dataset(definition: DatasetDefinition, paths: DatasetPaths) -> dict[str, pl.DataFrame]:
```

Comment:
it confuses me somehow why mypy didn't complain before about this typehint. Although I refactored and simplified some of the logic inside the function, the returned type didn't change.